routers/web/repo/issue.go

```diff
@@ -467,6 +467,9 @@ func issues(ctx *context.Context, milestoneID, projectID int64, isPullOption opt
 	ctx.Data["AssigneeID"] = assigneeID
 	ctx.Data["PosterID"] = posterID
 	ctx.Data["Keyword"] = keyword
+	if isShowClosed.Has() {
+		ctx.Data["IsShowClosed"] = isShowClosed.Value()
```

Comment:
I think since `IsShowClosed` has 3 states, it's better to change the template. the current implementation has only two states. `ctx.Data["IsShowClosed"] = false` is equal to not set the template variable.

Reply:
As far as I know when `state` query parameter is not setted (that also means `IsShowClosed` does not have value), issue list shows opened issues. 
In think in that case, `ctx.Data["IsShowClosed"] = false` doesn't seemed to error prone.
Is there are any edge cases that i'm missing?